packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -308,7 +308,7 @@ Build the phone number entry page that accepts the phone number. The user uses t
 
 </div>
 
-> **Note:** The SDK requires that the phone number be in the following format: `+#######`, including the beginning plus (+) sign. See [Data Requirements - Phone number](/docs/guides/oie-embedded-sdk-common/aspnet/main/#phone-number).
+> **Note:** The SDK requires that the phone number be in the following format: `+#######`, including the beginning plus (+) sign, for example, `+5551234567`.
```

Comment:
@brianduffield-okta perhaps all three notes could follow the format of this note?